frontend/tsconfig.json

```diff
@@ -23,7 +28,11 @@
     "*.ts",
     "*.tsx",
     "**/*.ts",
-    "**/*.tsx"
+    "**/*.tsx",
+    "*.js",
```

Comment:
This will make our linter complain about Typescript violations in all the non-Typescript (js/jsx) files, but I think that's probably OK if it encourages us to finish converting those pages to Typescript. : - )